IntegrationTests/Services/AWSS3IntegrationTests/S3ToggleUnsignedTests.swift

```diff
@@ -0,0 +1,45 @@
+//
```

Comment:
Please add:
- One more test case that has payload over the chunked threshold
- An interceptor that asserts that:
  - `Authorization` header isn't present as expected
  -  `x-amz-content-sha256` header values match expected values (`UNSIGNED-PAYLOAD` for single chunk request, and `STREAMING-UNSIGNED-PAYLOAD-TRAILER` for aws chunked request)

Reply:
I believe `Authorization` header is still expected to be there. We are still signing the request just not the body. Will do the rest

Reply:
That's true, and gotcha